icefoganalytics/elcc-data-management
# Fixup: Fix Casing of Variables Now That Everything Is Camelized

# Context

I switched that database side to using camelCasing.
As a result most of the UI broke, I'm pretty sure I've migrated everything over to using camelCase.

# Implemenation

Added some seeders for FundingSubmissionLine data.
I made a migration to swap the FundingSubmissionLineJson#value data to camelized as well.

# Testing Instructions

1. Boot the app via `dev up`

2. Check that the app complies, and that you can log in at http://localhost:8080.

3. Check that you can go to all the pages and things look the way they are supposed to.
# Context

I switched that database side to using camelCasing.
As a result most of the UI broke, I'm pretty sure I've migrated everything over to using camelCase.

# Implemenation

I made a migration to swap the FundingSubmissionLineJson#value data to camelized as well.

# Testing Instructions

1. Boot the app via `dev up`

2. Check that the app complies, and that you can log in at http://localhost:8080.

3. Check that you can go to all the pages and things look the way they are supposed to.
